src/reanimated2/js-reanimated/index.ts

```diff
@@ -31,8 +33,8 @@ export const _updatePropsJS = (
       [{}, {}]
     );
 
-    if (typeof component.setNativeProps === 'function') {
-      setNativeProps(component, rawStyles);
+    if (component.style !== undefined) {
```

Comment:
I remember that the else statement (`else if (Object.keys(component.props).length > 0)`) was prepared for handling some SVG's components. Did you check the `blame` if those particular components don't contain the `style` property? Just to be sure that we don't break something.

Reply:
Looks like the lines you've mentioned were added in #2197 (see [here](https://github.com/software-mansion/react-native-reanimated/blame/437cb56ab7c567ab3ef0a63482bdfb06e03f2e5f/src/reanimated2/js-reanimated/index.ts#L47)) so most likely it's all about basic Chrome Debugger support. As for animating SVG components, somehow you were right, fixed in ac60d65e7d8826ace33d298f797c7d7fa2aabe44